quinn-udp/src/unix.rs

```diff
@@ -794,21 +794,120 @@ pub(crate) const BATCH_SIZE: usize = 32;
 #[cfg(apple_slow)]
 pub(crate) const BATCH_SIZE: usize = 1;
 
+#[cfg(any(target_os = "linux", target_os = "android"))]
+mod linux {
```

Comment:
Nit: there's no point in making a separate module for this, just move the contents of this into the existing Linux `gso` module. Try to keep the additions in top-down order, so callers come before callees.